mart/callbacks/adversary_connector.py

```diff
@@ -67,6 +67,7 @@ def on_after_batch_transfer(self, pl_module, batch, dataloader_idx):
         adversary.to(pl_module.device)
 
         # Directly pass batch instead of assuming it has a structure.
-        batch_adv = adversary(batch=batch, model=pl_module)
+        adversary.fit(batch=batch, model=pl_module)
+        batch_adv = adversary.forward(batch=batch)
```

Comment:
Just call adversary?

Reply:
Fixed in 102ef988704fc97de0fb1dac38dcfd9ec5eb1e4c